app/src/main/java/at/irfc/app/ui/program/ProgramScreen.kt

```diff
@@ -55,14 +63,82 @@ fun ProgramScreen(
 
             eventListResource.data?.let { eventList ->
                 items(eventList, EventWithDetails::id) { event ->
-                    Text(
-                        text = event.title,
+                    Card(
+                        shape = RoundedCornerShape(20.dp),
+                        elevation = CardDefaults.cardElevation(
+                            defaultElevation = 10.dp
+                        ),
                         modifier = Modifier
-                            .padding(5.dp)
-                            .clickable {
-                                navController.navigate(ProgramDetailScreenDestination(event.id))
+                            .padding(bottom = 16.dp)
+                            .fillMaxWidth()
+                    ) {
+                        Row(
+                            modifier = Modifier
+                                .clickable {
+                                    navController.navigate(
+                                        ProgramDetailScreenDestination(event.id)
+                                    )
+                                }
```

Comment:
In my opinion the whole card schould be clickable - modifier must be applied to the card. Otherwise there will be a small area of the card which you can't click which is weird and also the ripple effect of the click will not fill the whole card